src/data/terraform.json

```diff
@@ -95,7 +96,7 @@
 		},
 		{
 			"iconName": "docs",
-			"name": "Intro",
+			"name": "Intro to Terraform",
```

Comment:
Also just an observation, the `/terraform` sidebar now has two "Intro" sections that both go to the "What is Terraform?" page: one at the top level and one under "Documentation". Is this intended?

Reply:
fwiw, i like the top level link placement more